MSAL/src/requests/MSALBaseRequest.m

```diff
@@ -252,7 +252,7 @@ - (BOOL)verifyTokenResponse:(MSIDAADV2TokenResponse *)tokenResponse
 
     if (![self.oauth2Factory verifyResponse:tokenResponse context:nil error:&msidError])
     {
-        if (error) *error = msidError;
+        if (error) *error = [self interactionRequiredErrorFromMSIDError:msidError];
```

Comment:
nit: name seems to indicate it will create interactionRequiredError no matter what. 
Although it might be more lines, but perhaps change so that this would return a BOOL and you create an interaction required error here?

Or change the name of the method,,to...... i don't know..... :p

Reply:
Makes total sense, will update :)